tailcfg/tailcfg.go

```diff
@@ -680,6 +680,10 @@ type Location struct {
 	// IATA, ICAO or ISO 3166-2 codes are recommended ("YSE")
 	CityCode string `json:",omitempty"`
 
+	// Latitude, Longitude are optional float64 polar coordinates of the city.
```

Comment:
I'm not sure if the godoc format is really ok with listing these together, but here's some suggested phrasing.

```suggestion
	// Latitude, Longitude are the geographical coordinates of the node.
	// No particular accuracy level is promised, the coordinates may simply be the center of the city or country.
```